### PR TITLE
Program

### PR DESCRIPTION
**Pull request by marcusshannon**
Tests for programs

### CODE REVIEW

**Comment by tkindy** [comment]
What's the status of this pull request?

**Comment by marcusshannon** [comment]
It's the tests for programs, I don't think anyone has reviewed them yet.

File: test/unit/programs.js [hunk 1]
```diff
@@ -0,0 +1,495 @@
+process.env.NODE_ENV = 'development';
```
Comment (tkindy) [inline test/unit/programs.js]: You don't need this line. The default NODE_ENV value is 'development'.

File: test/unit/programs.js [hunk 2]
```diff
@@ -0,0 +1,495 @@
...
+const programs = require('../../routes/programs');
+const seed = require('../../lib/utils').seed;
+const chai = require('chai');
+const assert = chai.assert;
+
+
```
Comment (tkindy) [inline test/unit/programs.js]: Remove this extra line (for styling consistency)

File: test/unit/programs.js [hunk 3]
```diff
@@ -0,0 +1,495 @@
...
+const program3 = {program_id: 3, site_id: 10, program_name: 'test program 3'};
+const program4 = {program_id: 4, site_id: 11, program_name: 'test program 4'};
+var allPrograms = [program1, program2, program3, program4];
+
+var programsDeleted = [program2, program3, program4];
+const newProgram = {program_id: 5, site_id: 5, program_name: 'new test program'};
+var programsNew = [program1, program2, program3, program4, newProgram];
+const updatedProgram = {program_id: 1, site_id: 1, program_name: 'updated program'};
+
+describe('GET', function(done) {
```
Comment (tkindy) [inline test/unit/programs.js]: Take a look at the [students tests](https://github.com/CS4500-AmericaSCORES/america-scores-backend/blob/master/test/unit/students.js). We should unify the way we describe our tests, and I think the way @jacquelineali described her tests is really great for a quick idea of what's going right/wrong.

**Comment by tkindy** [comment]
The build's failing. Can you get it to pass?

**Comment by marcusshannon** [comment]
Yeah, Jackie is gonna push a fix in utils then I'm good to go

**Comment by tkindy** [comment]
Okay, the develop build is now passing again. There's still some issues with your tests

**Pull request merged**